launch-or-update-azure.sh

```diff
@@ -30,10 +30,14 @@ function displayHelp {
             [REQUIRED] Url to connect to Redis
           -q, --redis-password <redis host password>
             [REQUIRED] Password connect to Redis
-          -e, --environment
-            The environment override file to apply. Default is 'all' (which won't apply anything). Other options are currently 'private' and 'public'.
+          -f, --file-overrides
+            A comma-separated list of file names. These are the names of override files to apply.
+            NOTE: these files are expected to be in the 'environments' directory
+            E.g: '-f public.yaml,local-docker.yml' will apply environments/public.yaml & environments/local-docker.yml 
```

Comment:
this is a bad example to give, as this script isn't yet meant for use when running against local docker. Probably best to use no-test-data.

Reply:
The example was supposed to demonstrate the differences in file extensions, but I take your point.